lib/loader.js

```diff
@@ -30,8 +39,8 @@ module.exports = function(content, map) {
 
 	processCss(content, map, {
 		mode: moduleMode ? "local" : "global",
-		from: loaderUtils.getRemainingRequest(this),
-		to: loaderUtils.getCurrentRequest(this),
+		from: loaderUtils.getRemainingRequest(this).split("!").pop(),
```

Comment:
What is the output of `loaderUtils.getRemainingRequest(this).split("!").pop()`? If I remember right it should be the absolute file path and therefore we could replaced it with `this.resourcePath` instead?